components/Block.json

```diff
@@ -0,0 +1,73 @@
+{
+  "hash": {
+    "title": "block hash",
+    "$ref": "./Data.json#/hash32"
+  },
+  "blockObject": {
+    "title": "block object",
+    "type": "object",
+    "required": ["transactions", "uncles"],
+    "allOf": [
+      {
+        "$ref": "./Header.json#/headerObject"
+      },
+      {
+        "$ref": "./Block.json#/transactionHash"
+      },
```

Comment:
this doesnt look right combining `headerObject` with an array of tx hashes.

it should be combining `type: 'object'`s together with the correct shape that you want to merge:

```
"allOf": [
      {
        "$ref": "./Header.json#/headerObject"
      },
      {
       "type: "object",
       "properties": {
           "transactions": {
              "$ref": "./Block.json#/transactionHash"
           }
       },
        {
       "type: "object",
       "properties": {
           "uncles": {
              "$ref": "./Block.json#/transactionHash"
           }
       }
]       
```

side note: the name `transactionHash` seems like an incorrect name for an array of transaction hashes.

also you can pass `true` to `eth_getBlockByNumber` and it returns the transactions filled out instead of just the hashes:


![image](https://user-images.githubusercontent.com/364566/127212050-362cd7ec-c6a8-4fe5-b732-dca512415167.png)

so it needs to take that into account too